docs: add StrongBrew to the trainer list

KwintenP opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
There is no link to the StrongBrew website
Issue Number: N/A


## What is the new behavior?
There is a link to the StrongBrew website

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

KwintenP committed:
docs(aio): add StrongBrew to the trainer list

mary-poppins commented:
You can preview e4cddd4 at https://pr24891-e4cddd4.ngbuilds.io/.

StephenFluin commented on aio/content/marketing/resources.json:
```diff
@@ -181,7 +181,7 @@
             "title": "Amexio Canvas Web Based Drag and Drop IDE by MetaMagic",
             "url": "https://amexio.tech/"
-          }	
+          }
```
Looks like we have an unrelated change here?

KwintenP replied:
As discussed further, the trailing tab can be removed.

StephenFluin reviewed (changes requested):
There's one unrelated change that needs to be removed.

Since you'll need to update the branch anyway, can you change docs: to `docs(aio):` in your commit message? If you still have the branch locally you can just make the changes and then `git commit -am --amend` to amend the commit and then `git push -f` to rewrite your branch. You shouldn't need another PR.

mary-poppins commented:
You can preview 1a6f680 at https://pr24891-1a6f680.ngbuilds.io/.

brandonroberts commented on aio/content/marketing/resources.json:
```diff
@@ -181,7 +181,7 @@
             "title": "Amexio Canvas Web Based Drag and Drop IDE by MetaMagic",
             "url": "https://amexio.tech/"
-          }	
+          }  
```
Remove the whitespace change

brandonroberts replied:
Sure

trotyl replied:
CC. @StephenFluin per https://github.com/angular/angular/pull/24891#discussion_r203389477

KwintenP replied:
Removed it.

KwintenP commented:
My apologies for the late response, I was on holiday.

Updated as requested. Removed the tab (which my editor did itself, hence the change). 

Thanks!

mary-poppins commented:
You can preview ef0403e at https://pr24891-ef0403e.ngbuilds.io/.
